apps/workers/sql-load/worker.go

```diff
@@ -28,6 +29,7 @@ type InfoURI struct {
 	Table        string            `uri:"table" required:"true"`      // insert table name i.e., "schema.table_name"
 	SkipErr      bool              `uri:"skip_err"`                   // if bad records are found they are skipped and logged instead of throwing an error
 	DeleteMap    map[string]string `uri:"delete"`                     // map used to build the delete query statement
+	DeleteCustom string            `uri:"delete_custom"`              // delete params statement is provided as a string value
```

Comment:
name? Should this be called Delete_Query or delete_statement or delete_sql instead?

Reply:
I like delete_sql